import/marc_local.properties

```diff
@@ -67,7 +67,18 @@
 
 # Uncomment and modify the following line to customize the creation of work keys
 # used to identify different record versions. The example below includes the
-# Finnish/Swedish characters in the list of characters to keep. See
+# Finnish/Swedish characters in the list of characters to keep.
+#
+# Parameters:
+#   uniform title fields (field spec)
+#   title fields (field spec)
+#   title fields with non-filing characters removed (field spec)
+#   author fields (field spec)
+#   characters to include (regex)
+#   characters to exclude (regex)
+#   transliterations
+#
+# See
 # https://unicode-org.github.io/icu/userguide/transforms/general/#icu-transliterators
 # for more information on the transliteration rules.
-#work_keys_str_mv = custom, getWorkKeys(130anp:730anp, 240anpmr:245abn:246abn:247abn, 100ab:110ab:111ac:700ab:710ab:711ac, "", "", ":: NFD; :: lower; a\U00000308>AE; o\U00000308>OE; a\U0000030A>AA; :: Latin; :: [:Nonspacing Mark:] Remove; :: [:Punctuation:] Remove; :: [:Whitespace:] Remove; :: NFKC; AE>ä; OE>ö; AA>å")
+#work_keys_str_mv = custom, getWorkKeys(130anp:730anp, 240anpmr:245abn:245abn:246abn:247abn, 240anpmr:245abn, 100ab:110ab:111ac:700ab:710ab:711ac, "", "", ":: NFD; :: lower; a\U00000308>AE; o\U00000308>OE; a\U0000030A>AA; :: Latin; :: [:Nonspacing Mark:] Remove; :: [:Punctuation:] Remove; :: [:Whitespace:] Remove; :: NFKC; AE>ä; OE>ö; AA>å")
```

Comment:
I notice that`245abn` is repeated twice in this example (`240anpmr:245abn:245abn:246abn:247abn`). Should the second parameter actually be `240anpmr:245abn:246abn:247abn` (unchanged from before)?